src/main/kotlin/no/nav/familie/ef/sak/service/steg/BeslutteVedtakSteg.kt

```diff
@@ -70,6 +74,25 @@ class BeslutteVedtakSteg(private val taskRepository: TaskRepository,
         }
     }
 
+    private fun opprettTaskForBehandlingsstatistikk(behandlingId: UUID, oppgaveId: Long?) {
+        val vedtak = vedtakService.hentVedtak(behandlingId)
+        val vedtakstidspunkt = behandlingshistorikkService.finnSisteBehandlingshistorikk(behandlingId, StegType.SEND_TIL_BESLUTTER)?.endretTid ?: error("Mangler behandlingshistorikk for vedtak")
```

Comment:
Burde ikke `vedtakstidspunkt` være det samme som endretTid for vedtak? Sånn att man ikke trenger å hente behandlingshistorikk?

Reply:
Jo, men det finnes ikke noen "endretTid" på et vedtak ennå. Kan legge inn "todo" på denne

Reply:
Ah 👍